tests/MoneyTest.php

```diff
@@ -365,4 +382,50 @@ public function modExamples()
             [1007, 10, '7'],
         ];
     }
+
+    public function aggregateExamples()
+    {
+        return [
+            [
+                [Money::EUR(5), Money::EUR(10), Money::EUR(15)],
+                null,
+                Money::EUR(5),
+                Money::EUR(15),
+                Money::EUR(30),
+                Money::EUR(10),
+            ],
+            [
+                [Money::EUR(-5), Money::EUR(-10), Money::EUR(-15)],
+                null,
+                Money::EUR(-15),
+                Money::EUR(-5),
+                Money::EUR(-30),
+                Money::EUR(-10),
+            ],
+            [
+                [Money::USD(-5), Money::EUR(-10), Money::EUR(-15)],
+                \InvalidArgumentException::class,
+                null,
+                null,
+                null,
+                null,
+            ],
+            [
+                [],
+                version_compare(PHP_VERSION, '7.0.0') >= 0 ? \ArgumentCountError::class : Error::class,
```

Comment:
This should rather be a separate test with a phpunit version requirement

https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.skipping-tests-using-requires